bin/requirements.txt

```diff
@@ -1,4 +1,4 @@
-docker==2.7.0
+docker==5.0.3
```

Comment:
Failing Buildkite build showed that `bb8 status` is broken (confirmed broken on prod) due to a bug with the docker package. No idea how long that's been the case, but updating seems to have fixed it.